pom.xml

```diff
@@ -91,33 +90,44 @@
         <guava.version>18.0</guava.version>
         <findbugs.version>3.0.1</findbugs.version>
         <commons-codec.version>1.10</commons-codec.version>
-        <spring.version>3.2.17.RELEASE</spring.version>
+        <spring.version>4.2.5.RELEASE</spring.version>
         <jetty.version>9.2.19.v20160908</jetty.version>
         <pax.cdi.version>0.12.0</pax.cdi.version>
         <validation.version>1.1.0.Final</validation.version>
         <commons-configuration.version>1.10</commons-configuration.version>
-        <servicemix.bundles.mina.version>1.1.7_6</servicemix.bundles.mina.version>
-        <qpid-jms-client.version>0.6.0</qpid-jms-client.version>
-        <proton-j.version>0.10</proton-j.version>
-        <netty-all.version>4.0.17.Final</netty-all.version>
-        <geronimo-jta_1.1_spec.version>1.1.1</geronimo-jta_1.1_spec.version>
-        <geronimo-jms_1.1_spec.version>1.1.1</geronimo-jms_1.1_spec.version>
+        <commons.fileupload.version>1.3.2</commons.fileupload.version>
+        <logback.classic.version>1.1.7</logback.classic.version>
+        <!--camel dependencies-->
+        <camel.version>2.18.0</camel.version>
+        <servicemix-specs.version>2.7.0</servicemix-specs.version>
+        <geronimo-jta-spec.version>1.1.1</geronimo-jta-spec.version>
+        <geronimo-jms-spec.version>1.1.1</geronimo-jms-spec.version>
+        <commons-lang.version>2.6</commons-lang.version>
```

Comment:
Looks like a few of these are duplicates. `commons-lang` and `commons-collections` are defined above.